### PR TITLE
Map change

### PR DESCRIPTION
**Pull request by Sheylos**
Moved sec departures checkpoint, expanded robotics and fixed arcade power issue

### CODE REVIEW

**Comment by optimumtact** [comment]
What is going on here?

**Comment by francinum** [comment]
The fuck is this

**Comment by WJohn** [comment]
WHAT HAPPENED

**Comment by TechnoAlchemisto** [comment]
what

**Review by optimumtact — APPROVED** [review]
This looks good to me

**Comment by TrustyGun** [comment]
um
what

File: config/admins.txt [hunk 1]
```diff
@@ -7,115 +7,11 @@
...
-bear1ake = Game Master
-CoreOverload = Game Master
-Jalleo = Game Master
-Anonus Tanir = Game Master
-FoxPMcCloud = Game Master
-Xhuis = Game Master
-Astralenigma = Game Master
-Tokiko1 = Game Master
-SuperSayu = Game Master
-Lzimann = Game Master
```
Comment (lzimann) [inline config/admins.txt]: how DARE you to remove me?!

**Comment by francinum** [comment]
THIS IS A GREAT IDEA

**Comment by ChangelingRain** [comment]
>Good job you memeing fuck

I think that covers it! We are not your downstream server. Please figure out github and try again.

**Comment by optimumtact** [comment]
@ChangelingRain

**Comment by Incoming5643** [comment]
<!doctype html>
<html>
<head>
    <title>Example Domain</title>

    <meta charset="utf-8" />
    <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type="text/css">
    body {
        background-color: #f0f0f2;
        margin: 0;
        padding: 0;
        font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
        
    }
    div {
        width: 600px;
        margin: 5em auto;
        padding: 50px;
        background-color: #fff;
        border-radius: 1em;
    }
    a:link, a:visited {
        color: #38488f;
        text-decoration: none;
    }
    @media (max-width: 700px) {
        body {
            background-color: #fff;
        }
        div {
            width: auto;
            margin: 0 auto;
            border-radius: 0;
            padding: 1em;
        }
    }
    </style>    
</head>

<body>
<div>
    <h1>Example Domain</h1>
    <p>This domain is established to be used for illustrative examples in documents. You may use this
    domain in examples without prior coordination or asking for permission.</p>
    <p><a href="http://www.iana.org/domains/example">More information...</a></p>
</div>
</body>
</html>

**Review by MrsTonedOne — COMMENTED** [review]
HOLY JESUS!
WHAT IS THAT?
WHAT
THE
FUCK
IS
THAT?

**Review by francinum — APPROVED** [review]
when the meme so supreme you let out a scream

**Review by ChangelingRain — CHANGES_REQUESTED** [review]
> Good job you memeing fuck

I think that covers it! We are not your downstream server. Please figure out github and try again.

**Review by lzimann — CHANGES_REQUESTED** [review]

File: config/admins.txt (hunk 1 above)
Comment (francinum) [inline config/admins.txt]: Just cleaning up some mistakes

**Comment by pudl** [comment]
just merge it and see what happens

File: tgstation.dme [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
-#include "_maps\tgstation2.dm"
+#include "_maps\metastation.dm"
```
Comment (MrStonedOne) [inline tgstation.dme]: At least it's not all bad, they know what map is better.

**Comment by JJRcop** [comment]
Sieracode/-tg-station#90

File: tgstation.dme (hunk 2 above)
Comment (optimumtact) [inline tgstation.dme]: BOX IS LIFE

Comment (francinum) [inline tgstation.dme]: And you never lived.

Comment (lzimann) [inline tgstation.dme]: \>meta
\>being better

**Review by Razharas — COMMENTED** [review]
image

**Comment by PKPenguin321** [comment]
holy fuck my sides

**Comment by Cheridan** [comment]
nothing to see here

move along

**Comment by AdamElTablawy** [comment]
what the fuck

**Comment by francinum** [comment]
wHAT HAVE YOU DONE

File: config/admins.txt (hunk 1 above)
Comment (optimumtact) [inline config/admins.txt]: still savage as fuck

**Pull request closed without merging**